src/state/actions/dispatchArIOInteraction.ts

```diff
@@ -58,8 +58,16 @@ export default async function dispatchArIOInteraction({
         let antProcessId: string = payload.processId;
 
         if (antProcessId === 'atomic') {
+          const state =
+            payload.state ||
+            (payload.targetId
```

Comment:
Seems odd here to check payload.targetId then later use payload?.targetId?. Can this be condensed down to:

```
 const state =
            payload.state ||
            createAntStateForOwner(
                  owner.toString(),
                  payload?.targetId?.toString(),
                );
                ```